source/content/drush-versions.md

```diff
@@ -54,21 +65,29 @@ Always use Drush 8 with Drupal 7 and Drupal 6 sites, as Drush 9 only works on Dr
 | Drush 7       | PHP 5.3.0+          |
 | Drush 8       | PHP 5.4.5+          |
 | Drush 9       | PHP 5.6.0+          |
+| Drush 10      | PHP 7.1.+           |
 
 See our guide on [Upgrading PHP Versions](/php-versions).
 
 ## Troubleshooting
 
-Sometimes even after updating the drush version in `pantheon.yml`, the correct version of drush is not called. This is usually caused by an outdated configuration file, `policy.drush.inc`, in your local `~/.drush` directory, overriding `pantheon.yml`. Remove the file, or comment out its contents, to resolve.
+Sometimes even after updating the Drush version in `pantheon.yml`, the correct version of Drush is not called.
+
+The Pantheon platform always prefers the site-local Drush or other local settings over the setting in `pantheon.yml`. Check for an outdated configuration file, `policy.drush.inc`, in your local `~/.drush` directory. Remove the file, or comment out its contents to resolve.
```

Comment:
Does `~/.drush` refer to the users' home directory, or the root of the codebase for the site? If the former, I don't understand how it could affect drush calls on the platform. If the latter, I'll update in my edits to make this clearer.

CC @greg-1-anderson

Reply:
Thanks! Resolved by dcf2a7ff5.